### DISCUSSION TITLE
Can't send big file (already using local telegram bot api)

### DISCUSSION THREAD

**Discussion by alchist** [aiogram/aiogram#1382] [Q&A] [answered]
Hello. I'm already using local telegram bot api and can download bigger files from user. But when I try to send file bigger than 50 mb it got stuck. 
Here is code I' m using:

 ```
file = FSInputFile(filePath)
message = await bot.send_video(userId, file)
```

It works with smaller files but with bigger I never got results.

**Comment by JrooTJunior** [comment]
You can only send large file from file system instead of using InputFile.

Just provide full path to the file as string into API method.

Note that this file should be accessible for the Bot API server by the same path that you provide in the API call.

**Comment by alchist** [comment]
So I should be able to just do something like 

`message = await bot.send_video(userId, <absolute file path>)`

It doesn't seems to work for some reason. I launched local bot api from admin, so it should have access to the file

**Reply by alchist** [reply, marked as answer]
I solved problem passing local URL to function
